### DISCUSSION TITLE
FlinkCDC postgres to doris时不支持表的schema变更

### DISCUSSION THREAD

**Discussion by tchivs** [apache/flink-cdc#3405] [Q&A]
在pg中为表添加一个字段并赋值，查看doris表并没有添加新列，找到[debezium-postgresql](https://debezium.io/documentation/reference/stable/connectors/postgresql.html)页面上说 **逻辑解码不支持 DDL 更改。这意味着连接器无法将 DDL 更改事件报告给使用者。** 难怪我在doris连接器的JsonDebeziumSchemaSerializer中接收不到任何DDL变更的消息。是确实不支持还是我操作有误，有人成功过吗。

**Comment by linmingze** [comment]
原理上不支持的，要做要用触发器记录ddl
你是sql 单表同步吗
